File: Conceptual_Code_In_Java/Command.java
import java.util.*;

class Command
{
	public static void main(String Arg[])
	{
		System.out.println("Total number of arguments are : "+Arg.length);
		
		System.out.println("Command line arguments are : ");
		for(int i = 0; i < Arg.length; i++)
		{
			System.out.println(Arg[i]);
		}
	}
}

File: Conceptual_Code_In_Java/Command.c
///////////////////////////////Command_Line_Argument_Program////////////////////////////////////

#include<stdio.h>
#include<stdlib.h>

int main(int argc, char *argv[])
{
	printf("Total number of command line Arguments are : %d\n",argc);
	
	for(int i = 0; i < argc; i++)
	{
		printf("%s\n",argv[i]);
	}
	
	return 0;
}